latexify/core.py

```diff
@@ -163,9 +167,44 @@ def visit_Compare(self, node):
 
     if isinstance(node.ops[0], ast.Eq):
       return lstr + '=' + rstr
+    # CHANGE:(20200801@1MLightyears) Add support for conditions for <,>,\ge,\le,\ne,is
+    elif isinstance(node.ops[0], ast.Gt):
+      return lstr + '>' + rstr
+    elif isinstance(node.ops[0], ast.Lt):
+      return lstr + '<' + rstr
+    elif isinstance(node.ops[0], ast.GtE):
+      return lstr + r'\ge ' + rstr
+    elif isinstance(node.ops[0], ast.LtE):
+      return lstr + r'\le ' + rstr
+    elif isinstance(node.ops[0], ast.NotEq):
+      return lstr + r'\ne ' + rstr
+    elif isinstance(node.ops[0], ast.Is):
```

Comment:
for "is"&"is not":
I tested several cases-the layout, the view, the font, of "is" and "is not", and found that what I really need may actually be "\in" and "\notin". Because normal papers would almost never have statements like "$if x is int$", but "$if x \in Z$". 
An approach is that the user manually define a variable like `Z=type('Z',(int,),{})` and then use ` if x is Z` to have the output "$if x is Z$". So maybe change "$is$" and "$is not$" to "$\in$" and "$\notin$" is better. 
I cannot decide this alone, so it's great to make some discussion for this.

Reply:
How is this going?

For the use-cases like `x is int`, the equivalent Python functionality is `isinstance(x, int)`. The Python's `is` is not a `belongs-to (\in)` operator: it checks if the two expressions represent the same instance, e.g., `1 is 1` is `True`. Also the Python's `in` is provided basically for collection types. If we want to implement the `\in` operator, I think we need to take two implementations: the Python's `in` and the pattern match replacement for `isinstance`. As the latter one may involve some garbages, it may be good to restrict acdeptable types (e.g., number types only).

If we provided some representation of `is`, I think the nearest one is maybe *isomorphic* (`\equiv`) though there's some differences between them.

Reply:
Oh I thought you mean not to change them at present.
OK, I will change the implementation of the `ast.Is` clause into `\equiv ` and delete the `ast.IsNot` clause. As for `isinstance`, that couldn't be processed in `visit_Compare` so maybe that might be implemented in the future.